jenkinsfiles/UB18_Check.groovy

```diff
@@ -0,0 +1,373 @@
+// Manages communication with github for reporting important status
+// events. Relies on Github integration plugin https://github.com/jenkinsci/github-plugin
+class GithubStatus {
```

Comment:
I see the same class in jenkinsfiles/RHEL_Check.groovy, is it possible to avoid the code duplication? The same question about other common parts.

Reply:
Hi Alexey, let me explain. Firstly we intend to make it to be a common class can be shared by different scripts. But that requires us to introduce a new folder under root path of the repo named "src" which saves common classes, required by Jenkins Groovy Syntax. In order to avoid misunderstanding about the repo and source code, we decided to write it into different scripts. I will investigate more about it but I think maybe currently there is no better way considering the requirement of our CI process for oneDPL.

Reply:
@DoyleLi, thank you for the explanation! I'm not experienced in Jenkins Groovy, so I wouldn't insist on resolving the duplication in this PR, but it makes sense to improve it in future in any case. Maybe you can consider some other Jenkins plugin for more clear configurations.